### PR TITLE
Cloning is no longer roundstart unless paid for

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request
Despite efforts made by another person's Synthflesh-cloner PR a long while ago, the Medbay meta has barely changed. Doctors still brainlessly dump people in cloning, except now they make sure to use plumbing to setup a Synthflesh factory beforehand.

Doctors still don't know tend wounds or even just synthflesh splashing before defibbing, which should always be the go-to rather than cloning. The next step to correct the glaring deficiency in our doctors competency is by making cloning no longer available roundstart.

Cloning machines have been replaced by empty frames on all maps. Cloning is still available through research, however now it has surgery research as a pre-requisite. Cloning boards now are also available via Cargo for emergency situations as well if Science isn't doing their job but you do however have a surplus of cash. Cargo cloning is likely to be especially useful for LRP players if Science or Medbay aren't doing their jobs with revival or cloning tech. 

## Why It's Good For The Game
Doctors learn the better, faster and less chemical-dependent form of revival finally. 

For patients it makes them spend less time in Medbay since in early game there's plenty of free doctors available for quick revival (which is faster than cloning + cryo for clone damage) and in late game there is T4 cloning machines available. 

For doctors they get a job to do instead of being replaced by a brainless two-click machine, for early game at least anyway.

## Changelog
:cl:
balance: Due to budget cuts CentCom has had to recall expensive cloning machinery from it's stations. They can still be obtained however through Science or Cargo. 
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
As a medbay main, I love this change.

**Comment by r1ks-iwnl** [comment]
This would probably end up with cloner actually having upgraded parts more often, husks would be a bit of a problem early but i do think it's a good change.

**Comment by LordSaladin** [comment]
this just buffs murderboning on golden

**Comment by Archanial** [comment]
The cloner is not a problem, the synthflesh factories are the actual problem.

**Comment by PowerfulBacon** [comment]
The problem isn't exactly with cloning, this is a bad PR and won't work without medical system unless it is reworked entirely.
The bigger issue with medical than cloning is the fact that the doctors are almost not at all needed and skills aren't required since all you need is a synthflesh patch dispenser and boom, doctors not needed.
The other thing is that at the moment, either doctors will spend half the round trying to clone a body that isn't worth reviving without cloning since the alternative methods to cloning are not effective.
Completely removing cloning doesn't fix medical, just shifts the problem. If you really want a medical system with more depth, cloning and surgery need to be altered so that rather than doing the same task and trying to balance to methods that have the same outcome is have the 2 different methods be better at different things and you use 1 in certain situations and 1 in the other. 

Edit: Another bad thing about doing it this way, is it's not a nerf as much as it's an annoyance that is going to be required at almost every round or you are going to be dead especially on low pop

Also
> Doctors learn the better, faster and less chemical-dependent form of revival finally.

It's not better, it's not faster.
It takes a ridiculously long time since you need to replace organs, heal their damage, do surgery to fix broken organs, then have the surgery which is quite expensive. Every since the change to how organs decay on dead bodies, surgery revival has been terrible and not worth it.

**Comment by ike709** [comment]
We don't want cobbymed.

**Comment by VictorVow** [comment]
> all you need is a synthflesh patch dispenser and boom, doctors not needed.

I agree synthflesh is a problem even for normal healing, I believe tg and other places have it give toxin damage as well. But even if synthflesh/plumbing was nerfed for cloning usage it would still be the case that cloning is available as a roundstart option, when the reality is there are extremely few cases where a patient can only be revived by cloning and not by revival surgery or defibbing. It's not necessary for it to exist roundstart at all. 

> It's not better, it's not faster.

It's better since you get to keep your augments, nanites, mindshield, dental implants and all other body modifications. It's faster since generally all a doctor has to do is tend wounds for 2 minutes then defib. In some cases they might need to do a coronary bypass also which takes an additional minute. In worst case scenarios they have to cut open your chest and replace several organs at once, but this still only takes an additional minute or two longer. Compare this to sitting in a cloner, then sitting in a cryo cell, then having to go get all your body modifications replaced. 

> It takes a ridiculously long time since you need to replace organs, heal their damage, do surgery to fix broken organs, then have the surgery which is quite expensive

When I was in my worst state before being revived a while ago all I had was the surface damage healed (2 mins), 3 organs replaced for their cybernetic equivalent (3 mins), then defibbed. It was far quicker than you're suggesting to the point I'm not sure if you've undergone revival surgery in a while. You're making it out as if organ decay is something that happens instantly and ignoring the fact that you can be made defibbable with just a few minutes work. 

How do you think people on /tg/ have survived without cloning for this entire time?

**Comment by ghost** [comment]
> We don't want cobbymed.

i do.

**Comment by KitsuneQueen** [comment]
I agree with Bacon. Cloning is a gear in the machine of problems that make up the current medical system. Synthflesh is the meta and has been for aeons. Medical doctors spend more time setting up factories and the alike so they don't have a job to do, then complain or get bored when no one comes to them for medical attention.

Cobbychem answers these issues with having chemicals have a downside, synthflesh dealing toxin damage for example and brute patches doing liver damage. These are good balancing that makes chems not the go-to answer but rather a temp fix to an issue a patient has.

Does this sound familiar? Sleepers in the medical foyer, auto surgeon, medical lathe, and synthflesh patches all where the public can reach. Because, medical is simple, not time consuming, and anyone can do it. I'm just happy people haven't figured out they can move cloning outside medical, too.

What are some solutions? Completely removing medical and put everything already outside where people can reach OR go the TG route, making medical more active.

"But if I die at the start of the round, it will take longer to be revived/If a traitor kills me or murderbones, then I might come back!" Tough luck, you aren't entitled to live on this space station. If you're an antags target, you aren't entitled to live and he isn't expected to fail.

If you do happen to die out of your own choices, then it's your own actions that led to your death. Sometimes you die, deal with it. Or, sometimes you die due to matters outside your control, there's a hint in the game for this—

"Sometimes you die due to matters outside your control, accept this."

**Comment by Sinmero** [comment]
> The problem isn't exactly with cloning, this is a bad PR and won't work without medical system unless it is reworked entirely.
> The bigger issue with medical than cloning is the fact that the doctors are almost not at all needed and skills aren't required since all you need is a synthflesh patch dispenser and boom, doctors not needed.
> The other thing is that at the moment, either doctors will spend half the round trying to clone a body that isn't worth reviving without cloning since the alternative methods to cloning are not effective.
> Completely removing cloning doesn't fix medical, just shifts the problem. If you really want a medical system with more depth, cloning and surgery need to be altered so that rather than doing the same task and trying to balance to methods that have the same outcome is have the 2 different methods be better at different things and you use 1 in certain situations and 1 in the other.
> 
> Edit: Another bad thing about doing it this way, is it's not a nerf as much as it's an annoyance that is going to be required at almost every round or you are going to be dead especially on low pop
> 
> Also
> 
> > Doctors learn the better, faster and less chemical-dependent form of revival finally.
> 
> It's not better, it's not faster.
> It takes a ridiculously long time since you need to replace organs, heal their damage, do surgery to fix broken organs, then have the surgery which is quite expensive. Every since the change to how organs decay on dead bodies, surgery revival has been terrible and not worth it.

I added od threshold for synthflesh but here is a problem - splashing with synthflesh does not add the reagent to the bloodstream so the od check doesnt work properly. Is there any way to detect splash action and the amount? Might be a good idea to remove healing from splashing at all. Or might just make it ingestive medicine instead.

**Comment by ivanmixo** [comment]
`Doctors still don't know tend wounds or even just synthflesh splashing before defibbing, which should always be the go-to rather than cloning.`


Why? What is wrong with just cloning someone?

**Comment by KitsuneQueen** [comment]
> `Doctors still don't know tend wounds or even just synthflesh splashing before defibbing, which should always be the go-to rather than cloning.`
> 
> Why? What is wrong with just cloning someone?

That is the problem, you just clone them. Imagine if there were no doors in the way of cloning, it was public. Would you REALLY need a medical team to revive someone if anyone could just click a button and revive someone? It makes medical redundant. Look at my original point, MDs spend more time trying to lose their job rather than keep it. Unlike chad me, who hides medkits round start and takes the surgical tools so people HAVE to come to me. MHAWAWAWAWAW~

On TG, this isn't a problem. MDs have a job to do and a pretty intense one at that, too. TGs new wound system is great fun and means you can spend more time with one patient and interact with them, which is a good part of medical in my books -- befriending your patients.

Yet, on Bee I don't find myself talking with many people. Not only won't they talk, but they are most likely children that can't hold a conversation for ten minutes without walking away.

I got off topic:
Cloning bad, takes no skill. Da end.

**Comment by TotallyNotChief** [comment]
we tg now boys

**Comment by ike709** [comment]
Cloning people as a crutch replacement for doing actual medical is cringe yes, but cobbychem is still stinky and not going to happen.

**Comment by St0rmC4st3r** [comment]
> `Doctors still don't know tend wounds or even just synthflesh splashing before defibbing, which should always be the go-to rather than cloning.`
> 
> Why? What is wrong with just cloning someone?

Memories and disrespect of medic mains.

**Comment by VictorVow** [comment]
> Cloning people as a crutch replacement for doing actual medical is cringe yes, but cobbychem is still stinky and not going to happen.

This isn't cobbychems, meds are still robust as hell for healing living people. This only addresses _roundstart_ cloning, which is even less necessary with the surplus of Medbay storage chemicals available and bored doctors.

**Comment by ivanmixo** [comment]
You guys are underestimating the average ss13 player's level of knowledge.

From my experience on tg medical is often just carried by one doctor who knows what he's doing while the other ones just watch.


You overestimate how fun it is to watch bars go up round after round.

**Comment by eLeCtrOssSnake** [comment]
Apparently I don't have a fancy tag to say that I don't like as ike can, nevertheless I can't.
Me no like.

**Review by Cenrus — APPROVED** [review]

**Comment by ike709** [comment]
> This only addresses roundstart cloning, which is even less necessary with the surplus of Medbay storage chemicals available and bored doctors.

Lol med storage usually gets completely raided within the first 20 minutes.

**Comment by VictorVow** [comment]
On LRP sure but not MRP. Budget cards will also get raided on LRP so there there's cash to use Cargo to order replacement medikits, synthflesh and a cloning suite pack. 

And if its argued "but LRP players won't do that!!1" then what makes you think the existing cloner will ever get refilled or upgraded past it's current 1 clone maximum current capability. Either way the cloner isn't used without some small bit of effort.

**Comment by ivanmixo** [comment]
> On LRP sure but not MRP. Budget cards will also get raided on LRP so there there's cash to use Cargo to order replacement medikits, synthflesh and a cloning suite pack.
> 
> And if its argued "but LRP players won't do that!!1" then what makes you think the existing cloner will ever get refilled or upgraded past it's current 1 clone maximum current capability. Either way the cloner isn't used without some small bit of effort.

> And if its argued "but LRP players won't do that!!1" then what makes you think the existing cloner will ever get refilled or upgraded past it's current 1 clone maximum current capability.

Do you even play on LRP?

**Comment by FortSumter** [comment]
Hey Hey, LRP main here. I've suggested for a while now if you want to downgrade cloning, just give cloning the prototype cloner at roundstart than the normal one. That way it still serves a purpose (getting players back into the game, or providing food/genetics brain-dead copies) and if people want to come back as themselves they can research the actual cloner.

When it comes to medical in general, no one wants to do it. We might get some chemists and geneticists, maybe even a paramedic, but medical is the second least desired department behind security. This is due to the above reasons mentioned by others that it's simply BORING to watch bars slowly go up and then have to do it again when it fails. There's nothing to do as a medical doctor at roundstart besides set up cryo, and nothing else to do for the rest of the game if no one gets injured. You just sit there as a medical doctor with a thumb up your ass... or begin causing trouble themselves as is often the case. 

There's also a rather accurate stigma against medical that they are all assholes out to fuck you rather than help you, so you might as well do it yourself. I personally rather break into medical and heal myself than trust a doctor not to morphine me; god help anyone who actually requires surgery. A legit helpful and/or trained doctor or chemist is very rare in a server filled with people only interested in themselves and are always regarded with suspicion.

In short, removing the cloner really doesn't solve anything. Make medical more interesting instead, and maybe hold medical staff to a higher level of responsibility like we do with security so people will actually trust them.

**Comment by Ratvarr** [comment]
i have seen defibs used, however its only with the metagangs that know _everything_ that do it, since most often when you die
that's it, no more round, if your LUCKY you might get cloned but that's just about it
this might be good for mrp, not for lrp

**Comment by Moccha-Bee** [comment]
shite pr don't merge.

**Comment by jlsnow301** [comment]
I agree and disagree. You may be competent enough to ask for a more difficult department, but the vast majority of medical isn't, that's if they even remember to wrench the o2 canisters in place for cryo. Medical is largely no different than engineering: SM is pretty easy to set up, but rarely does anyone do it and it usually results in a shuttle call. 

If the main complaint here is doctors needing a job and cloning being too drag-and-scan, I think nerf should instead cause more complex problems instead of being outright removed. More like cloning damage and negative traits that can only be operated on.

For instance: You can clone, but there'd be a chance the person needs organ replacements, punches people all shift, or spawns some horror similar to _The Fly_. In a previous suggestion, I thought cloning SSD players should spawn zombies instead of catatonics. My philosophy here is chaos first, still is, instead of just removing some aspect of the game

**Comment by CarpStation** [comment]
I'd want this for carp station - My other idea to solve this issue was tier 1 cloner would have issues. I.e limbs/organs missing

**Comment by hotimep** [comment]
can we port permadeath next

**Comment by Chessus** [comment]
TL:DR.- Make the cloner only work on bodies/brains that have been dead for x amount of time (15 mins is the time the defib stops working).If you don't want to wait, you do use the brain but suffer consequences.
-Make stasis bed and Operating computers tech available at roundstart
-Education is the main issue. It's up to  the CMO and others to communicate to newer players about the alternatives.  


As a med main the problem is education. Newbie docs who don't ask/read the wiki will do this. I did this until I was told that other revival methods were faster such as TW/synthflesh with a coronary bypass and defib. That Formaldehyde (now in starting Epipens) stop organ decay and more stasis beds aren't required. I think the onus should be on the CMO and vet docs to communicate on :M channel. 

Yes the cloner should be a last resort or in a medbay full of bodies. At roundstart your only options are Cloner or defib (with tw/synthflesh and organ surgery if required)
Any other means require interdepartmental co-operation which we know is a pain:
-Revival Surgery won't be researched. It also required a Operating computer for the option to appear. 
-Chem can make Strange Reagent but this requires a chem is isn't making bombs and holy water from Chaplin/Cargo.
-Stasis beds and Operating computers need to be researched. 

As for synthflesh, I love it and don't want it changed. It's fun to use, convenient and isn't laborious especially for minor injuries. When you're a doc in a disaster situation you'll understand it's usefulness. You need access to chemistry(which a doc doesn't have) or plumbing equipment to make(which isn't always available outside the chem lab). Cause the roundstart cloner uses 90u and the cmo's bottle is 100u, that's only 2 clones you can do without Synthflesh production. Frequently The CMO(Who has access to the engineering lather btw) and science won't upgrade the cloner so you're stuck with the tier 1 version for a while. 

I agree with you idea, I think this isn't the way to solve it though. think other alternatives would be:
- To have the parts of the cloner be available in the room and would need setting up.
- To have the cloner only work on bodies that have been dead close to or after the Defib timer(15 mins, any longer defib doesn't work)

In a perfect world the cloner wouldn't be such an issue but with the nature of the game, many different players and varying experience, part of the challenge is to overcome this.

**Comment by PowerfulBacon** [comment]
Going to be honest, this likely isn't going to be merged unless there are some more changes rather than just removing it from the maps and replacing it with a machine frame.
 - This doesn't solve the issue of medical being shallow, if a doctor wants to do surgery on someone they can.
 - This doesn't make medical less shallow or more enjoyable for the doctors, it adds something they either have to do roundstart or the bodies are going to end up piling up.
 - People shouldn't be forced to use 'the better way', if you are going to change cloning you should have it as an alternative to surgery based revivals and have it better at some things, but worse at others.
